ontology/pg.ttl

```diff
@@ -0,0 +1,102 @@
+# baseURI: http://biohackathon.org/resource/pg
```

Comment:
I would be happy to have all of this in the vg file. If we have a separate one we need to ask @ktym to set up the file copying to the biohackathon website. perhaps we can change the html generation to credit your work as well?

Reply:
Sure, then I would merge all data format into `vg.ttl`.